src/app/shared/home-component/home.component.html

```diff
@@ -130,6 +134,26 @@ <h3>
           </a>
         </sky-column>
       </sky-row>
+      <sky-row>
+        <sky-column screenSmall="4">
+          <a routerLink="/components" class="screenshot screenshot-info">
+            <img src="~/assets/home/stache.png" alt="Stache" />
+          </a>
+        </sky-column>
+        <sky-column screenSmall="2">
+          &nbsp;
+        </sky-column>
+        <sky-column screenSmall="6">
+            <div class="info">
+              <h3>
+                Documentation - Stache
```

Comment:
I'd change this header to "Docs as Code" since that's the recommendation I got for the SKY whitepaper. (If you don't like that, I'd at least delete "- Stache" from the header. Or worst-case, change the hyphen to a long dash.)

By the same token, I'd tweak what you have in the paragraph that follows to emphasize the docs as code angle: 
"The Stache component library provides open source components built on SKY UX to generate consistent, user-focused technical documentation websites with the same tools that you use to write code. <a href="https://developer.blackbaud.com/stache">Learn more</a>."

Reply:
Agreed.